src/search/landmarks/landmark_factory_reasonable_orders_hps.cc

```diff
@@ -73,7 +76,7 @@ void LandmarkFactoryReasonableOrdersHPS::approximate_reasonable_orders(
                     for (const auto &p : node2.parents) {   // find parent
                         LandmarkNode &parent = *(p.first);
                         const EdgeType &edge = p.second;
-                        if (parent.disjunctive)
+                        if (parent.get_landmark().disjunctive)
```

Comment:
Why ```p``` to ```parent```, wouldn't ```parent_nodes``` make much more sense?

Reply:
```parent_node``` certainly makes sense, but I don't like ```parent``` because it doesn't give away that its both the node and the edge. As this will change with the new design anyway, I suggest to stay with (the obviously poorly chosen) ```p``` for now.